R/equivalencebayesianindependentsamplesttest.R

```diff
@@ -152,7 +152,10 @@ EquivalenceBayesianIndependentSamplesTTest <- function(jaspResults, dataset, opt
   if (ready)
     equivalenceBayesianIndTTestTable$setExpectedSize(length(options$variables))
   
-  equivalenceBayesianIndTTestTable$addFootnote(.equivalenceGetIntervalMessage(options$lowerbound, options$upperbound))
+  message <- gettextf("I ranges from %1$s to %2$s", 
```

Comment:
I don't think these changes are correct.
https://github.com/jasp-stats/jaspEquivalenceTTests/commit/7457abeb09c8a0cfee4bf31178347981206440c8 made sure to remove the duplication of these lines

Reply:
@TimKDJ Ah I see, I was boobytrapped. These fixes were only done on the jaspEquivalence Module repo not in stable. 
I do not think it is necessary to backport them to stable so will I close this PR?

Reply:
I'm not sure if we want the table renaming though, @Jillderon?

Reply:
No changes are not needed, this PR can be closed!